test/parallel/test-repl.js

```diff
@@ -142,7 +142,7 @@ function error_test() {
       expect: prompt_unix },
     // But passing the same string to eval() should throw
     { client: client_unix, send: 'eval("function test_func() {")',
-      expect: /\bSyntaxError: Unexpected end of input/ },
+      expect: /^\bSyntaxError: Unexpected end of input/ },
```

Comment:
@vsemozhetbyt 
1. Well I tried that. On introducing a `^` in [L193](https://github.com/adityaanandmc/node/blob/52798349d7f77493b9d36372eb9dc7ca24a5e582/test/parallel/test-repl.js#L193) which uses strict mode, I get the `Octal literals are not allowed in strict mode` error as below: (Same case as in other lines you had mentioned )
```
Unix data: "(function() { \"use strict\"; return 0755; })()\n 
^^^^\n\nSyntaxError: Octal literals are not allowed in strict mode.\n\nnode via Unix socket> node via Unix socket> ", expecting /^SyntaxError: Octal literals are not allowed in strict mode/
assert.js:42
```

2. Yeah makes sense. Will do that! Can I go ahead and remove the`\b` even where I can't introduce a `^` cause of strict mode ?